src/ai/ai.cpp

```diff
@@ -385,30 +399,30 @@ void initialize_ai_tech_weights(sys::state& state) {
 	for(auto t : state.world.in_technology) {
 		float base = 1000.0f;
 		if(state.culture_definitions.tech_folders[t.get_folder_index()].category == culture::tech_category::army)
-			base *= 1.5f;
+			base *= 3.0f;
 
 		if(t.get_increase_building(economy::province_building_type::naval_base))
-			base *= 1.1f;
+			base *= 9.0f;
```

Comment:
really, *nine* times more weight to increasing naval base level? Nothing should have more weight than the weight given to techs that increase the research point modifier. I should also point out that only the relative values matter. Increasing all the weights won't really matter; only the relative weightings matter in most cases